04-protocol-messages.md

```diff
@@ -126,29 +148,31 @@ The following convenience types extend 'BOLT #1' fundamental types:
 
 `core_arbitrating_setup` sends the `lock (b)`, `cancel (d)` and `refund (e)` arbitrating transactions from Bob to Alice, as well as Bob's signature for the `cancel (d)` transaction.
 
- 1. type: 33710 (`coresetup`)
+ 1. type: 33710 (`core_arbitrating_setup`)
  2. data:
+    - [`swap_id`: `swap_id`] The swap unique identifier
     - [`u16`: `lock_len`]
     - [`lock_len * byte`: `lock`] The arbitrating `lock (b)` transaction
     - [`u16`: `cancel_len`]
     - [`cancel_len * byte`: `cancel`] The arbitrating `cancel (d)` transaction
     - [`u16`: `refund_len`]
     - [`refund_len * byte`: `refund`] The arbitrating `refund (e)` transaction
     - [`u16`: `cancel_sig_len`]
-    - [`cancel_sig_len * byte`: `cancel_sig`] The `Bc` `cancel (d)` signature
+    - [`cancel_sig_len * byte`: `cancel_sig`] The `Bc` `cancel (d)` arbitrating `signature` type
```

Comment:
Not convinced about the usage of `type` from here on

Reply:
I should add a reference to 07, where types for cryptographic primitives are defined or drop the `type` mention.

Why I did that: ECDSA signatures is `>64 bytes` where Schnorr signatures are `=64 bytes`, their types are different.

Overall I think `type` here is not critical so I'll drop it.